dev/com.ibm.ws.jaxws.clientcontainer_fat/fat/src/com/ibm/ws/jaxws/clientcontainer/fat/FATSuite.java

```diff
@@ -25,4 +28,8 @@
                 WsBndServiceRefOverrideTest.class
 })
 public class FATSuite {
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                    .andWith(FeatureReplacementAction.EE8_FEATURES());
```

Comment:
This change is simply the easiest way to test a the jaxwsClient-2.3 feature, but will not be delivered once I've verified  that the runtime update is successful in a Personal Build.